test/stripe/account_test.exs

```diff
@@ -1,49 +1,26 @@
 defmodule Stripe.AccountTest do
   use ExUnit.Case
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
 
   setup_all do
-    Stripe.Accounts.delete_all
+    use_cassette "Stripe.AccountTest/setup", match_requests_on: [:query, :request_body] do
```

Comment:
Unfortunately, by default `ExVCR` doesn't have these two bits on. Thus I had to specify it manually every time `use_cassette` is used.

Having ExVCR check on `query` and `request_body` is specifically useful in cases when multiple instances of the same entity type are created within a single test, multiple accounts (see line 61 and 62).